cdap-app-templates/cdap-etl/cdap-etl-proto/src/main/java/io/cdap/cdap/etl/proto/v2/ArgumentMapping.java

```diff
@@ -48,11 +59,21 @@ public String getTarget() {
     return target;
   }
 
+  /**
+   * @return The identifiers of properties from the triggering pipeline.
+   */
+  @Nullable
+  public TriggeringPipelineId getTriggeringPipelineId() {
+    return pipelineId;
+  }
+
   @Override
   public String toString() {
     return "ArgumentMapping{" +
       "source='" + getSource() + '\'' +
       ", target='" + getTarget() + '\'' +
+        ", triggeringPipelinePropertyId='" +
+        (pipelineId == null ? "" : pipelineId.toString()) +  '\'' +
```

Comment:
don't need to do this, just do `+ pipelineId` like it's done for source and target.

Reply:
`pipelineId` is a Class instead of String like source and target.  Guess have to do a null check...

Reply:
if it has a toString() method java will use that, you don't need to do this manually

Reply:
gotcha! that's a good point, thanks!